ci-operator/config/openshift/external-dns-operator/openshift-external-dns-operator-release-1.2.yaml

```diff
@@ -0,0 +1,153 @@
+base_images:
+  operator-index:
+    name: redhat-operator-index
+    namespace: ci
+    tag: v4.9
+build_root:
+  from_repository: true
+images:
+- dockerfile_path: Dockerfile
+  to: external-dns-operator
+operator:
+  bundles:
+  - as: external-dns-operator-bundle
+    base_index: operator-index
+    dockerfile_path: Dockerfile.bundle
+  substitutions:
+  - pullspec: quay.io/openshift/origin-external-dns-operator:latest
+    with: pipeline:external-dns-operator
+releases:
+  initial:
+    integration:
+      name: "4.14"
+      namespace: ocp
+  latest:
+    integration:
+      include_built_images: true
+      name: "4.14"
+      namespace: ocp
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: verify
+  commands: |
+    export GOLANGCI_LINT_CACHE=$(mktemp -d)
+    make verify
+  container:
+    from: src
+- as: unit
+  commands: |
+    export XDG_CACHE_HOME=$(mktemp -d)
+    make test
+  container:
+    from: src
+- as: e2e-aws-operator
+  steps:
+    cluster_profile: aws
+    dependencies:
+      OO_INDEX: ci-index-external-dns-operator-bundle
+    env:
+      OO_CHANNEL: stable-v1
```

Comment:
Why isn't this `stable-v1.2`? Doesn't this fetch the latest package from the `stable-v1` channel? Currently, it's set to 1.2?, but if we upgrade to 1.3, wouldn't this 1.2 job incorrectly use version 1.3?

Reply:
You are right, we should use the minor channel here, good catch!